lib/rs/Cargo.toml

```diff
@@ -11,9 +11,8 @@ exclude = ["Makefile*", "test/**", "*.iml"]
 keywords = ["thrift"]
 
 [dependencies]
-byteorder = "~1.2.1"
-integer-encoding = "~1.0.4"
-log = "~0.3.8"
-threadpool = "~1.7.1"
-try_from = "~0.2.2"
-
+ordered-float = "0.5"
+byteorder = "1"
```

Comment:
Is 1.X sufficient or does this need to be "1.2"

Reply:
No - you should be able to leave it to 1. There's no need to pin it to 1.2. But why was the ordered-float crate added?

Reply:
Generated code (https://github.com/apache/thrift/blob/3e5a1d9497062e2664a038daa668c3af954f72aa/compiler/cpp/src/thrift/generate/t_rs_generator.cc#L555) uses ordered_float, which means anyone that wants to use thrift also has to bring in ordered_float. Rust crates typically handle this by bringing in the dependency, and re-exporting it as necessary, which is the approach I took here. 

Bringing in thrift alone will now work for generated code, no additional crates are required.